src/sidebar/store/modules/groups.ts

```diff
@@ -18,12 +18,16 @@ export type State = {
   groups: Group[];
   /** ID of currently selected group. */
   focusedGroupId: string | null;
+
+  /** Members of currently selected group */
+  focusedGroupMembers: GroupMember[] | null;
```

Comment:
Since only members for the current group are stored, this means we'll be reloading members each time you switch back and forth. That's OK to start with, as long as we ensure that this is cleared and in-flight requests are canceled when switching groups, but in future we might want to cache members from groups you've selected in the current session.

Reply:
I'd prefer not to do client-side caching for now, as it could lead to things changing in the server and having to find a way to invalidate the cache remotely, or risk having outdated lists of users until you reload.

For annotations we do not do caching, for example. Every time the group changes we fetch all annotations for that group. Although it's true that group members will usually change less frequently than annotations, and annotations are loaded in one go.

Reply:
That's fine, we can skip caching for an initial implementation.